src/components/seoLayout/components/Layout.js

```diff
@@ -0,0 +1,63 @@
+import path from 'path'
```

Comment:
Looks good so far. @charlesjavelona  

I think you should call this SEO and use it as `<SEO/>` in addition to two other things:
- we have another "SEO" component. I'd recommend removing that if we don't need it.
- We have another component called Layout in this app. See if you can remove it or merge whatever is required from it, into this component.

Reply:
I think the old Layout component is where we're getting the default Univjobs logo from if we didn't include a `image` prop, because it's loaded before this new SEO layout component. That's a good thing. I'd take a look at it though to see if it can be worked into the new one.

It's at `layouts/index.js`.

Reply:
I renamed it to a folder called seo and the actual layout to SeoLayout. Didn't want to confuse the folder and file together.